.github/scripts/check_icon_pr.py

```diff
@@ -40,8 +45,8 @@ def main():
             svg_err_msg = check_svgs(svgs)
 
         err_msg = []
-        if devicon_err_msg != "":
-            err_msg.append(devicon_err_msg)
+        if devicon_err_msg != []:
+            err_ms.extend(devicon_err_msg)
```

Comment:
Fix a reference error.
```suggestion
            err_msg.extend(devicon_err_msg)
```